notebook/static/notebook/js/keyboardmanager.js

```diff
@@ -25,24 +25,32 @@ define([
          * @class KeyboardManager
          * @constructor
          * @param options {dict} Dictionary of keyword arguments :
-         *    @param options.events {$(Events)} instance 
+         *    @param options.events {$(Events)} instance
          *    @param options.pager: {Pager}  pager instance
          */
         this.mode = 'command';
+        this.view = 'tree';
```

Comment:
The keyboard interaction will change I think, but in time.  I think while we are still maintaining this branch, we might as well consider allowing this feature to be merged.  I'm of a similar mindset to @Carreau and am all for the addition of his and @svurens command palette, despite the refactor that is happening.